src/storage/txn/actions/acquire_pessimistic_lock.rs

```diff
@@ -69,7 +69,7 @@ pub fn acquire_pessimistic_lock<S: Snapshot>(
     // If any of `should_not_exist`, `need_value`, `need_check_existence` is set,
     // it infers a read to the value, in which case max_ts need to be updated to
     // guarantee the linearizability and snapshot isolation.
```

Comment:
Comments should be updated too.

Reply:
I thought it's necessary last week but I cannot come up with any counter example if max_ts not updated now. I'll remove this change from this PR, and add it back if it's confirmed to be necessary in the future.